src/Compilers/Test/Resources/Core/NetFX/Minimal/mincorlib.cs

```diff
@@ -2,8 +2,8 @@
 
 namespace System
 {
-	public class Object {}
-    public struct Byte {}
+    public class Object { }
```

Comment:
I will modify the PR so that I don't need to modify the shared mincorlib. This will allow me to better control what is there and what is missing. 
Thanks